src/functions.ts

```diff
@@ -17,13 +18,12 @@ function printName(name: string): void {
  */
 function printAge(birthYear: number): number {
   // write your code here
-
-  return -1; // replace -1 to what you see is fit
+console.log(2025 - birthYear)
```

Comment:
This will work today. It will work tomorrow. I will work until December 31 2025. What happens after that?